ddtrace/contrib/langchain/patch.py

```diff
@@ -51,6 +64,15 @@ def get_version():
     return getattr(langchain, "__version__", "")
 
 
+# After 0.1.0, implementation split into langchain, langchain_community, and langchain_core.
+# We need to check the version to determine which module to wrap, to avoid deprecation warnings
+# ref: https://github.com/DataDog/dd-trace-py/issues/8212
+LANGCHAIN_VERSION = parse_version(get_version())
+SHOULD_USE_LANGCHAIN_COMMUNITY = LANGCHAIN_VERSION >= (0, 1, 0)
+BASE_LANGCHAIN_MODULE = langchain_community if SHOULD_USE_LANGCHAIN_COMMUNITY else langchain
```

Comment:
BASE_LANGCHAIN_MODULE allows us to patch some `langchain_community` and `langchain` objects in the same way. Over the next few versions the structure of both packages might start to differ significantly.  Would it be easier to maintain an integration with minimal coupling? 

I am fine with doing this as a follow up

Reply:
Yes, this is true. Will add this as a next step for the planned refactor work. Thanks!